docs/how-to/troubleshoot_deployment_issues.md

```diff
@@ -0,0 +1,168 @@
+# Troubleshoot the deployment issues
+
+This guide provides step-by-step troubleshooting actions to remediate deployment issues. We hope you don't need this guide. If you encounter an issue and aren't able to address it via this guide, please raise an issue [here][Bug Report].
+
+## 1. Terraform failed to deploy the Charmed Aether SD-Core because of configuration mismatch
+
+### Symptoms
+
+Terraform may fail to deploy the Charmed Aether SD-Core modules. When the `terraform apply -auto-approve` command is run, the deployment fails with the wrong integration endpoints or the wrong object attributes etc. as following:
```

Comment:
```suggestion
The `terraform apply -auto-approve` command fails with the wrong integration endpoints or the wrong object attributes:
```

Reply:
This case is removed as it does not provide a solution.